sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Generated/Models/Resource.cs

```diff
@@ -63,75 +55,25 @@ public Resource(Kind kind, string location, string id = default(string), string
         partial void CustomInit();
 
         /// <summary>
-        /// Gets the resource identifier.
+        /// Gets fully qualified resource ID for the resource. Ex -
+        /// /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}
         /// </summary>
         [JsonProperty(PropertyName = "id")]
         public string Id { get; private set; }
 
         /// <summary>
-        /// Gets the resource name.
+        /// Gets the name of the resource
         /// </summary>
         [JsonProperty(PropertyName = "name")]
         public string Name { get; private set; }
 
         /// <summary>
-        /// Gets the resource type.
+        /// Gets the type of the resource. E.g.
+        /// "Microsoft.Compute/virtualMachines" or
+        /// "Microsoft.Storage/storageAccounts"
         /// </summary>
         [JsonProperty(PropertyName = "type")]
         public string Type { get; private set; }
 
-        /// <summary>
-        /// Gets or sets the kind of the service. Possible values include:
```

Comment:
This seems like a breaking change

Reply:
@markcowl this changes is coming from the common-types referenced by the Private Link. "$ref": "../../../../../common-types/resource-management/v1/privatelinks.json#/definitions/PrivateEndpointConnectionListResult". That's the reason I renamed the model definition in swagger spec from Resources -> ServiceResource.

Reply:
link to the PR approved by ARM team: https://github.com/Azure/azure-rest-api-specs-pr/pull/2046